staging/src/k8s.io/api/README.md

```diff
@@ -1 +1,21 @@
-This repo is still in the experimental stage. Shortly it will contain the schema of the API that are served by the Kubernetes apiserver.
+# api
+
+Schema of the external API types that are served by the Kubernetes API server.
+
+## Purpose
+
+This repo is still in the experimental stage.
```

Comment:
Let's remove this sentence.

Reply:
Done! 👍 Wonder if you can remove the description from the repo https://github.com/kubernetes/api? The "Soon-to-be the canonical location of the Kubernetes API definition." part is misleading a bit.

Reply:
Can you create an issue and assign it to @lavalamp ?